src/sorted_multi_dict.jl

```diff
@@ -469,13 +469,14 @@ function Base.show(io::IO, m::SortedMultiDict{K,D,Ord}) where {K,D,Ord <: Orderi
 end
 
 """
-    similar(sc)
+    empty(sc)
```

Comment:
Will `?empty` return three copies of this docstring now?

Reply:
Hah, yeah. Preexisting condition with `similar` though.

Reply:
Ah I see